embedded/tbtree/tbtree.go

```diff
@@ -1615,7 +1665,6 @@ func (n *innerNode) updateOnInsertAt(key []byte, value []byte, ts uint64) (n1 no
 	}
 
 	n._ts = ts
-	n.mut = true
```

Comment:
Why we don't mark here that the node was mutated ?

Reply:
because only mutated nodes are updated. A mutated node is created when trying to modify a non-mutated one (a copy is created in mutated state)